docs/add-ons/presentation.md

```diff
@@ -2,11 +2,11 @@
 title: "Presenting your addon"
 ---
 
-If you decide to share your add-on to the public, paying attention to details is recommended. Of course, your add-on should have a proper name and description, but Home Assistant also gives you some other tools to present your add-on even nicer.
+If you decide to share your add-on to the public, paying attention to details is recommended. Of course, your add-on should have a proper name and description, but Home Assistant also gives you some other tools to present your add-on even more nicely.
```

Comment:
Side note: This paragraph could use a rewrite.

Reply:
Done 9f73ec0